app/controllers/vendor_api/applications_controller.rb

```diff
@@ -13,19 +13,28 @@ def show
   private
 
     def serialized_application_choices_data
-      json_data = get_application_choices_for_provider_since(since: since_param).map do |application_choice|
-        ApplicationPresenter.new(version_number, application_choice).serialized_json
-      end
-
-      %({"data":[#{json_data.join(',')}]})
+      MultipleApplicationsPresenter.new(version_number, get_application_choices_for_provider_since(since: since_param), pagination_params).serialized_applications_data
     end
 
     def get_application_choices_for_provider_since(since:)
       application_choices_visible_to_provider
         .where('application_choices.updated_at > ?', since)
-        .find_each(batch_size: 500)
-        .sort_by(&:updated_at)
-        .reverse
+        .order('application_choices.updated_at DESC')
+    end
+
+    def pagination_params
+      {
+        since: since_param.iso8601,
+        page: params[:page],
+        per_page: params[:per_page],
+        api_version: params[:api_version],
+        url: url_for(
+          controller: params[:controller],
+          action: params[:action],
+          host: request.host,
+          api_version: params[:api_version],
```

Comment:
I think we might have access to `version_number` in the presenters through the version concern?

Reply:
Is there a preference for this? Would have to append a 'v' to it with version_number?
![image](https://user-images.githubusercontent.com/47917431/152387153-52f06025-af0f-4f4e-bcfe-459b68599336.png)